.swiftlint.yml

```diff
@@ -8,3 +8,6 @@ opt_in_rules:
 
 nesting:
   type_level: 4
+
+identifier_name:
+  allowed_symbols: "_"
```

Comment:
Beginning underlines are common for `@usableFromInline internal` stuff in Swift. Or generally for adding another layer of access control where performance is the bigger issue. It generally states: "You might see this symbol, but please refrain from using it unless you know what you're doing."